features/backend/metadata/metadata_customization.feature

```diff
@@ -1,75 +1,75 @@
 @metadata @ui
 Feature: Metadata customization
-  In order to customize metadata
-  As a store owner
-  I want to have user-friendly metadata customization form
+    In order to customize metadata
+    As a store owner
+    I want to have user-friendly metadata customization form
 
-  Background:
-    Given store has default configuration
-    And I am logged in as administrator
+    Background:
+        Given store has default configuration
+          And I am logged in as administrator
 
-  Scenario: Showing customization form
-    When I am customizing metadata
-    Then I should see metadata customization form
+    Scenario: Showing customization form
+         When I am customizing metadata
+         Then I should see metadata customization form
 
-  Scenario: Creating page metadata without Twitter metadata
-    Given I am customizing metadata with identifier "FOO-BAR"
-    When I fill in "Title" with "Lorem ipsum"
-    And I fill in "Keywords" with "sylius, ecommerce"
-    And I press "Save changes"
-    Then I should be on the page of metadata container with id "FOO-BAR"
-    And I should see the following metadata:
-      | Title       | Lorem ipsum       |
-      | Keywords    | sylius, ecommerce |
-      | Description | <empty>           |
-      | Twitter     | <empty>           |
+    Scenario: Creating page metadata without Twitter metadata
+        Given I am customizing metadata with identifier "FOO-BAR"
+         When I fill in "Title" with "Lorem ipsum"
+          And I fill in "Keywords" with "sylius, ecommerce"
+          And I press "Save changes"
+         Then I should be on the page of metadata container with id "FOO-BAR"
```

Comment:
We shouldn't align the steps to the right, they should be like natural sentences aligned to left like before that change.